feature: make position of actions configurable (#3459)
filamentphp/filament: packages/tables/docs/05-actions.md

]
```

> ## Position of row actions

Reviewer: ```suggestion
## Position
```

> By default, the row actions are rendered after all cells. To change the default postition, you may use:

Reviewer: ```suggestion
By default, the row actions are rendered in the final cell. To change the default position, you may use:
```
